src/components/Calculator.js

```diff
@@ -1,39 +1,44 @@
+import React, { useState } from 'react';
 import './calculator.css';
-import PropTypes from 'prop-types';
+import calculate from '../logic/calculate';
 
-function Calculator(props) {
-  const { onClick } = props;
+function Calculator() {
+  const [state, setState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (event) => {
+    setState(calculate(state, event.target.textContent));
+  };
```

Comment:
- [ ] The multiplication functionality of your app is currently not working. Please see the error image below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/220971620-382a5a24-4046-405c-afcf-24cf873afe1d.png)